document/core/syntax/values.rst

```diff
@@ -145,6 +145,19 @@ Conventions
 
 * The meta variable :math:`z` ranges over floating-point values where clear from context.
 
+.. index:: ! simd, fixed-width, vector
+   pair: abstract syntax; packed simd number
+.. _syntax-simd:
+
+Fixed-Width SIMD
+~~~~~~~~~~~~~~~~
+
+Fixed-width SIMD data represents packed integers, or floating-floating point values. The packed values of a given SIMD value must all be of the same type, these packed values can be any of the signed, unsigned integer types, and floating-point types ocurring in this proposal.
```

Comment:
nit: maybe "... must be all of the same type, which can be any of the ... ". It is probably worth mentioning the types explicitly here too.

Reply:
Fixed, and added a sentence about the explicit types.